bugsnag-android-core/src/main/java/com/bugsnag/android/ErrorTypes.kt

```diff
@@ -0,0 +1,29 @@
+package com.bugsnag.android
+
+class ErrorTypes(
+
+    /**
+     * Sets whether [ANRs](https://developer.android.com/topic/performance/vitals/anr)
+     * should be reported to Bugsnag. When enabled, Bugsnag will record an ANR whenever the main
+     * thread has been blocked for 5000 milliseconds or longer.
```

Comment:
This comment should probably be amended now that we are handling the signal rather than checking the main thread. Possibly fix in master and merge up?

Reply:
I plan on updating all the Javadoc once changes are complete on the integration branch and docs have been written for the docs site. As discussed IRL, leaving this as-is for now